x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authc/jwt/JwtStringClaimValidatorTests.java

```diff
@@ -36,7 +36,7 @@ public void testClaimIsNotString() throws ParseException {
     }
 
     public void testClaimIsNotSingleValued() throws ParseException {
-        final String claimName = randomAlphaOfLengthBetween(3, 8);
+        final String claimName = randomAlphaOfLengthBetween(10, 18);
```

Comment:
These new values are chosen to be consistent with what get used in the first test method.